test/WormholeL1GovernorMetadataBridge.t.sol

```diff
@@ -31,7 +31,8 @@ contract L1GovernorMetadataBridgeTest is Constants, WormholeRelayerBasicTest {
   }
 
   function setUpTarget() public override {
-    l2GovernorMetadata = new WormholeL2GovernorMetadata(L2_CHAIN.wormholeRelayer);
+    l2GovernorMetadata = new WormholeL2GovernorMetadata(L2_CHAIN.wormholeRelayer, msg.sender);
+    vm.prank(l2GovernorMetadata.owner());
```

Comment:
For some reason we need to do this. I logged `msg.sender` and it was the same as the owner

Reply:
Ahh ok sorry, reviewed in the wrong order, so I guess disregard my previous comments. Seems pretty odd though. I don't get it! Any thoughts @wildmolasses?